gulp/build.js

```diff
@@ -1,10 +1,20 @@
 const conf = require('../conf');
 
-const { task, parallel, src, dest } = require('gulp');
+const { task, series, parallel, src, dest } = require('gulp');
 const { join } = require('path');
 
 const rename = require('gulp-rename');
 const pegjs = require('gulp-pegjs');
+const ts = require('gulp-typescript');
+const tsProject = ts.createProject('tsconfig.json');
+
+task('build-typescript',
```

Comment:
build classes from typescript as part of the build pipeline.

Reply:
Noted, thank you